R/constants.R

```diff
@@ -168,6 +168,10 @@ DEFAULT_ELECTRIC_EFFICIENCY <- 0.33
 
 ELECTRICITY_INPUT_FUELS<- c("biomass", "coal", "gas", "refined liquids")
 
+#Default emissions coefficients to replace NA values in LA102.en_emiss_CDIAC (kGC/GJ)
```

Comment:
Space after `#` please. If these constants are used _only_ in LA102, probably move their definitions there.